cdap-app-fabric/src/main/java/io/cdap/cdap/master/environment/k8s/MasterEnvironmentMain.java

```diff
@@ -107,10 +107,18 @@ public static void doMain(String[] args) throws Exception {
         cConf.addResource(new File(options.getExtraConfPath(), "cdap-site.xml").toURI().toURL());
         sConf.addResource(new File(options.getExtraConfPath(), "cdap-security.xml").toURI().toURL());
       }
+      File cConfFile = new File("cConf.xml");
```

Comment:
Why we have two different ways to add cConf xml? There is one above using the extra conf path?

Reply:
I think the one above (using extra conf path) is unnecessary - removed it. should be sufficient to use localized cConf.xml